DOM.js

```diff
@@ -0,0 +1,26 @@
+document.addEventListener('DOMContentLoaded', function(){
```

Comment:
There is a single responsibility principle that states a method should do only one thing. Your event listener is doing too many things. Create other methods to handle different parts. Your event listener should just call one method. create other methods to do other parts